src/ClientData/ScopeStatsCollectionTest.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock-actions.h>
+#include <gmock/gmock-matchers.h>
+#include <gmock/gmock.h>
```

Comment:
Only the latter of the three should be included.

Reply:
Done